packages/@react-aria/toast/src/useToast.ts

```diff
@@ -10,17 +10,16 @@
  * governing permissions and limitations under the License.
  */
 
+import {chain} from '@react-aria/utils';
```

Comment:
I forgot to add `onRemove` to an action button when `shouldCloseOnAction`. I just added, but wondered if I should chain `onClose` and `onRemove` and pass that to the action button and close button instead of coding them slightly differently separately.

Reply:
I think it is fine as is